.github/ISSUE_TEMPLATE.md

```diff
@@ -8,11 +8,18 @@
 
 [this should explain **why** the current behaviour is a problem and why the expected output is a better solution.]
 
+**Note**: We receive a lot of issues on our GitHub tracker, so it is very possible that your issue has been posted before.
+Please check first before submitting so that we do not have to handle and close duplicates!
+
+**Note**: Many problems can be resolved by simply upgrading `pandas` to the latest version. Before submitting, please check
+if that solution works for you. If possible, you may want to check if `master` addresses this issue, but that is not necessary.
```

Comment:
I personally would leave out this one. It is fully correct, but we have to be aware that the longer the text in here, the less people will read it. I would rather put it in our contributing docs and clearly link to that as a guide on how to submit an issue